NX remote viewfinder information

Nikolas-LFDesigns opened the pull request:
You called-I'm here, op-co.de blog post about NX 300 RVF comments in one commit.

ge0rg commented:
Thanks very much! I was just going to suggest adding this to the wiki and not to the source code repository... but I'm glad to have it documented outside of my blog either way, so thanks! Keep it on!

Nikolas-LFDesigns commented:
moved into wiki. Added subsequent notice about it in readme (please correct if I've wrecked a chaos here :P).
smp_* will stick for now, they are in "source code" state for now, compilation follows ;)
